person.rb

```diff
@@ -11,6 +12,7 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
```

Comment:
- [ ] @Al-Amin-Khan-Shakil you did undoubtedly amazing job, but you missed a very small thing while writing the code, you have added the `instance` variable for adding `rentals` but you didn't provide the window for the user to `use` the `filter` to retrieve the data or **add** more rental data. So to resolve the problem, you just need to add the `getter` and `setter` for the **rentals**.